changelog/unreleased/add-setttings-to-monorepo

```diff
@@ -0,0 +1,28 @@
+Enhancement: Add the settings service
```

Comment:
File name s/setttings/settings/

But probably it makes no difference to anything. It is the content that matters for the changelog generation, and that is fine.